config/config.yaml

```diff
@@ -0,0 +1,27 @@
+# ODPF Stencil Configuration
```

Comment:
what if we rename this to `config.sample.yaml`?

Reply:
No value add IMO. `config.yaml` is clear and concise.

Reply:
I can see a warning in comments to tell users this is a sample config whereas if we just call it `sample` it will be a little more obvious?

Reply:
One reason to use just config is that I am thinking to provide a `config` command later which will use this file to init config for users instead of users looking and trying to find these files. In that case, the sample will give the wrong impression. WDYT?